sciris/sc_fileio.py

```diff
@@ -797,6 +797,9 @@ class Failed(object):
     
     def __init__(self, *args, **kwargs):
         pass
+    
+    def __setstate__(self, state):
+        pass
```

Comment:
@RomeshA I wasn't sure about this...

Reply:
@cliffckerr what happens if you don't include it?

Reply:
@RomeshA Not sure, now that we fixed the bugs with tzinfo, I don't have a test case that actually results in a `Failed` object :see_no_evil: